source/postprocess/geoid.cc

```diff
@@ -931,9 +1015,16 @@ namespace aspect
       {
         prm.enter_subsection("Geoid");
         {
+          // This parameter doesn't exist anymore (incl. spelling error). TODO: throw error message when used.
           prm.declare_entry("Include the contributon from dynamic topography", "true",
                             Patterns::Bool(),
                             "Option to include the contribution from dynamic topography on geoid. The default is true.");
+          prm.declare_entry("Include the contribution from surface topography", "true",
+                            Patterns::Bool(),
+                            "Option to include the contribution from surface topography on geoid. The default is true.");
+          prm.declare_entry("Include the contribution from CMB topography", "true",
+                            Patterns::Bool(),
+                            "Option to include the contribution from CMB topography on geoid. The default is true.");
```

Comment:
This is our chance to improve the naming scheme of the input variables. Here is my suggestion: `Include surface topography contribution` and `Include CMB topography contribution`.